Specs/Number/Arabic/NumberRangeModel.json

```diff
@@ -189,7 +188,7 @@
         "Text": "فوق ١٠٠٠ وتحت ١٥٠٠",
         "TypeName": "numberrange",
         "Resolution": {
-          "value": "-10,001,500"
```

Comment:
Same, can we add another spec case that outputs a range with negative?

Reply:
Sure! will add into next iteration.